desc/backend.py

```diff
@@ -73,6 +76,13 @@
     while_loop = jax.lax.while_loop
     vmap = jax.vmap
     bincount = jnp.bincount
+    from functools import partial
+
+    import jax
+    import jax.numpy as jnp
+    import jaxlib
+    from jax import config as jax_config
```

Comment:
i think these imports are either unnecessary or already imported above

Reply:
Ok.

Reply:
Removed.

Reply:
the lines are still there I think?